Lucas work

lnewcomer1-zz opened the pull request:
Hey i created a js function to resize the two images we have. made a variable called $cellphone that is 360 px wide. I did one breakpoint modification at $cellphone size to make the navbar words little smaller and the padding/margin a little smaller at the cellphone size and changed your padding-left in your breakpoint max-width $small in your "div.left  a" to 20px instead of 10px just becasue when you resize the screen to cellphone size that makes the title more centered which looks better I think. Also added the footer with the  diffferent social network icons. There still needs to be some resizing to be done between the cellphone size to bigger sizes when it comes to the navbar just so the transitioning of the window resizing will looks cleaner and more fluid as do the motion of resizing the width, but other than that I will continue to work on this.  I just thought I would create a pull request as of right now, so we can make sure this github process is working smoothly between both of us.

snewcomer commented:
Perfect!  Nice job.  I like it.  So now watch that single page design, b/c this site will somewhat resemble this and I am sure you will come up with ideas to improve this index skeleton.  However, what we are going to do is make an Angular application (there are also courses on Google's Angular if you want to watch it, but not a big deal at this point).  It basically goes from one page to another (Home, About, Contact) without refreshing. We will also use SCSS (not CSS) and Bootstrap. 

Next step: watch the sing page app and add these to the head section and below the footer and before the end of the body tag with the appropriate stylesheet (css) and script tags (js).  This allows us to get these libraries through the web rather than having the files loaded from our local file system.  Basically can replace <link href=...> with these.

//maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css
//maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js
find jquery and angular hosted libraries:
https://developers.google.com/speed/libraries/devguide

LucasNewcomer committed:
none

LucasNewcomer committed:
Resized images, got footer and breakpoint $cellphone created

Conflicts:
	builds/development/css/style.css
	builds/development/index.html
	components/sass/_modules.scss
	css/style.css
